apps/orgs/actions.py

```
@@ -134,7 +132,11 @@ def handle_managed_user_exception(org_id):
             'Error while creating Workato Workspace org_id - %s in Fyle %s',
             org.id, exception.message
         )  
-    
+
     except Exception:
         error = traceback.format_exc()
-        logger.error(error)
+        logger.info(
```

we shouldn't siltently ignore exception, this should be logger.error. Ideally we shouldn't reach this block